ui/apps/platform/cypress/integration/compliance-enhanced/complianceEnhancedDashboard.test.js

```diff
@@ -27,4 +37,32 @@ describe('Compliance Dashboard', () => {
         cy.get('th[scope="col"]:contains("Failing Controls")');
         cy.get('th[scope="col"]:contains("Last Scanned")');
     });
+
+    it('should visit scan configurations scheduling from the left nav', () => {
+        visitComplianceEnhancedScanConfigsFromLeftNav();
+
+        cy.title().should('match', getRegExpForTitleWithBranding('Scan Schedules'));
```

Comment:
Outside the scope of integration tests, should all these page titles include Compliance?

Left nav second level link is only **Scan Schedules** but page heading and title should have **Compliance**